docs/docusaurus/credentials/revocation.md

```diff
@@ -0,0 +1,89 @@
+# JWT credential revocation
+
+Identus implements revocation mechanism of JWT credentials according to [Bitstring Status List v1.0](https://www.w3.org/TR/2023/WD-vc-status-list-20230427/). It is an open standart that enables Identus to verify revocation status of any credential which implements revocation mechanism using the same specification.
+
+## Overview
+
+Every credential will contain the property `credentialStatus`, which will look like this
+
+```json
+"credentialStatus": {
+    "id": "http://localhost:8080/prism-agent/credentials/status/3#94567"
+    "type": "StatusList2021Entry",
+    "statusPurpose": "revocation",
+    "statusListIndex": "94567",
+    "statusListCredential": "http://localhost:8080/prism-agent/credentials/status/3"
+  },
+```
+
+* `type` will always be `StatusList2021Entry`
+* `statusListCredential` is a publically accessable URL that resolves a status list credential, it will look like this:
+```json
+{
+  "proof" : {
+    "type" : "DataIntegrityProof",
+    "proofPurpose" : "assertionMethod",
+    "verificationMethod" : "data:application/json;base64,eyJAY29udGV4dCI6WyJodHRwczovL3czaWQub3JnL3NlY3VyaXR5L211bHRpa2V5L3YxIl0sInR5cGUiOiJNdWx0aWtleSIsInB1YmxpY0tleU11bHRpYmFzZSI6InVNRll3RUFZSEtvWkl6ajBDQVFZRks0RUVBQW9EUWdBRUNYSUZsMlIxOGFtZUxELXlrU09HS1FvQ0JWYkZNNW91bGtjMnZJckp0UzRQWkJnMkxyNEQzUFdYR2xHTXB1aHdwSk84MEFpdzFXeVVHT1hONkJqSlFBPT0ifQ==",
+    "created" : "2024-03-23T16:45:50.924279Z",
+    "proofValue" : "ziKx1CJPKLy4U9kMmVzYct5xztq4oHRLPgMpAjh95zQxzBZorhLFmhZ85UPixJoQbaqkVaygLBnLARyxgGJGFNKFggaPSXHgJuG",
+    "cryptoSuite" : "eddsa-jcs-2022"
+  },
+  "@context" : [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/vc/status-list/2021/v1"
+  ],
+  "type" : [
+    "VerifiableCredential",
+    "StatusList2021Credential"
+  ],
+  "id" : "http://localhost:8080/prism-agent/credential-status/27526236-3836-4061-9867-f69314e258b4",
```

Comment:
@shotexa, I see there are two different URLs:
1. in the line 15: /credential/status/3
2. in the current line: /credential-status/{uuid}
Are both URLs valid and resolvable?